src/js/components/FileInput/stories/Multiple.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Grommet, FileInput } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const Example = () => {
+  return (
+    <Grommet full theme={grommet}>
+      <Box fill align="center" justify="start" pad="large">
+        <Box width="medium">
+          <FileInput
+            multiple
```

Comment:
The second item for the drop is being opened immediately instead of being added to the loaded components.
I wasn't able to add more than one file.

Reply:
should be fixed now